packages/bot-web-ui/src/components/dashboard/dashboard-components/index.scss

```diff
@@ -0,0 +1,178 @@
+    }
+}
+.dashboard {
+    &__container {
+        border: none;
+        max-width: 100%;
+        background: var(--general-main-1);
+        width: 100%;
+
+        .dc-tabs {
+            &__active-line {
+                background-color: var(--general-main-1);
+                transition: none;
+            }
+
+            &__active {
+                background: var(--general-main-1);
+            }
+
+            &__list {
+                background: var(--general-section-6);
+                justify-content: start;
+            }
+
+            &__item {
+                border-radius: 1rem 1rem 0px 0px;
+                height: 4.8rem;
+                width: 14.4rem;
+                display: flex;
+                justify-content: center;
+                align-items: center;
+                svg {
+                    width: 1.6rem;
+                    height: 1.6rem;
+                    padding-right: 0;
+                    margin-right: 0.8rem;
+                }
+            }
+            &__dashboard-cards-wrapper {
+                display: flex;
+            }
+
+            &__dashboard-card {
+                margin-right: 4.2rem;
+                display: flex;
+                flex-direction: column;
+
+                svg {
+                    margin-bottom: 0.8rem;
+
+                    &:hover {
+                        transition: all 100ms ease-in;
+                        box-shadow: rgba(0, 0, 0, 0.35) 0px 5px 15px;
+                        border-radius: 5px;
+                        font-weight: 700;
+                    }
+                }
+
+                span {
+                    text-align: center;
+                }
+
+                &:hover {
+                    cursor: pointer;
+                }
+                &:last-child {
+                    margin-right: 0;
+                }
+                &-image {
+                    background: var(--general-section-6);
+                }
+                &-label {
+                    font-size: 14px;
+                    font-weight: 400;
+                    line-height: 20px;
+                }
+
+                img {
+                    margin-bottom: 0.8rem;
+                }
+            }
+
+            &__content-group {
+                display: flex;
+                flex-direction: column;
+                justify-content: center;
+                align-items: center;
+                height: calc(100vh - 20rem);
+                margin-top: 1rem;
+
+                .heading {
+                    font-size: 20px;
+                    font-weight: 700;
+                    line-height: 30px;
+                    margin-bottom: 16px;
+                }
+
+                .description {
+                    font-size: 16px;
+                    font-weight: 400;
+                    line-height: 24px;
+                    margin-bottom: 5rem;
+                }
+            }
+        }
+    }
+}
+
+.side_bar_container_none {
```

Comment:
this naming is not standard, it should be .side-bar-container-none

Reply:
fixed ✅